apps/playground/src/app/components/text-editor/text-editor/text-editor.component.ts

```diff
@@ -54,9 +54,11 @@ export class TextEditorComponent implements OnInit {
     'alignment',
     'indentation',
     'undo-redo',
-    'link',
+    'link'
```

Comment:
I'm seeing this and other formatting changes unrelated to your feature which makes me think you may not have the Prettier extension installed in your IDE. Since the repo contains a Prettier config, it should be as simple as installing the extension and then saving the file to auto-format it using our rules.

Reply:
Installed Prettier and ran it!